bin/i18n/sync-out.rb

```diff
@@ -139,24 +140,30 @@ def restore_redacted_files
       translated_path = original_path.sub("original", locale)
       next unless File.file?(translated_path)
 
-      if original_path.include? "course_content"
+      if original_path == 'i18n/locales/original/dashboard/blocks.yml'
+        # Blocks are text, not markdown
+        RedactRestoreUtils.restore(original_path, translated_path, translated_path, plugins, 'txt')
```

Comment:
Where is `plugins` declared?

Is it okay this doesn't have the `blockfield` anymore? Or maybe it does, I'm not sure where `plugins` is.

Reply:
oh, woops! Good catch. Not sure why this didn't cause problems in my manual testing, I'll take a closer look at that

Reply:
the whole "changes.json" approach I think still does make the most sense from a functional standpoint, but boy does it sometimes make debugging a pain.

The reason I didn't catch this in my manual testing is because when I was running the sync out I didn't have any `blocks.yml` files in my local changes.json, so this code never got executed. I have since confirmed that it does indeed work now!